fdbclient/FDBTypes.h

```diff
@@ -1192,4 +1192,24 @@ struct ReadBlobGranuleContext {
 	bool debugNoMaterialize;
 };
 
+// Store metadata associated with each storage server
+struct StorageMetadataType {
+	constexpr static FileIdentifier file_identifier = 732123;
+	uint64_t createdTime; // comes from Platform::timer_int()
+	bool expireNow;
+	StorageMetadataType() : createdTime(0), expireNow(false) {}
+	StorageMetadataType(uint64_t t, bool expireNow = false) : createdTime(t), expireNow(expireNow) {}
+
+	// To change this serialization, ProtocolVersion::StorageMetadata must be updated, and downgrades need
+	// to be considered
+	template <class Ar>
+	void serialize(Ar& ar) {
+		serializer(ar, createdTime, expireNow);
+	}
+
+	std::string getCreatedTimeStr() const {
```

Comment:
given routine is converting time to seconds; better to name function as:
"getCreationTimeInSecStr()" ?

Reply:
I think this function converts a timestamp to a time representation of the GMT format. Maybe `getCreationTimeGMTStr()` is better?